tests/integration/pyramid_tests.py

```diff
@@ -33,12 +33,39 @@ class TestException(Exception):
     pass
 
 
+class ControlFlowException(Exception):
+    pass
+
+
+class ControlFlowException2(Exception):
```

Comment:
Not the best names but was running out of ideas. Definitely happy to take suggestions.